fmriprep/cli/run.py

```diff
@@ -27,6 +27,13 @@
   * Run identifier: {uuid}.
 """.format
 
+try:
+    import indexed_gzip
+    import nibabel
+    nibabel.arrayproxy.KEEP_FILE_OPEN_DEFAULT = 'auto'
+except ImportError:
+    pass
```

Comment:
You can just do `nibabel.arrayproxy.KEEP_FILE_OPEN_DEFAULT = 'auto'`, without checking imports. All that does is set the default to "use indexed_gzip if available, otherwise don't change behavior".